data/preshpi.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Precious Egwuenu",
+  "type": "personal",
+  "bio": "Frontend Developer and Content Creator",
+  "avatar": "https://github.com/preshpi.png",
```

Comment:
hey after the current update `avatar` is no longer needed. You can delete this line.
 ```suggestion
  
```